public_suffix_list.dat

```diff
@@ -15755,4 +15755,8 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+// frii.site : https://www.frii.site
```

Comment:
Please fix [sorting](https://github.com/publicsuffix/list/wiki/Guidelines#sort-your-submission-correctly-important) based on the guideline.